File: sample/javaSample/src/main/java/jp/co/geniee/samples/gnad/banner/BannerMenuActivity.java
package jp.co.geniee.samples.gnad.banner;

import android.content.Intent;

import jp.co.geniee.samples.BaseMenuActivity;
import jp.co.geniee.samples.MenuItem;

public class BannerMenuActivity extends BaseMenuActivity {


    @Override
    protected MenuItem[] getListViewContents() {
        MenuItem[] menuItem = {
                new MenuItem("Single Banner", "Classic banner ads", new Intent(this, SingleBannerDemoActivity.class)),
                new MenuItem("Multiple Banners", "Multiple banner ads", new Intent(this, MultipleBannerDemoActivity.class))
        };

        return menuItem;
    }
}

File: sample/UseLocalLibrary/GNAdSamples/kotlinSample/src/main/java/jp/co/geniee/samples/banner/BannerMenuActivity.kt
package jp.co.geniee.samples.banner

import android.content.Intent
import jp.co.geniee.samples.BaseMenuActivity
import jp.co.geniee.samples.MenuItem

class BannerMenuActivity : BaseMenuActivity() {
    override fun getListViewContents(): Array<MenuItem> {
        return arrayOf(
                MenuItem("Single Banner", "Classic banner ads", Intent(this, SingleBannerDemoActivity::class.java)),
                MenuItem("Multiple Banners", "Multiple banner ads", Intent(this, MultipleBannerDemoActivity::class.java))
        )
    }

}
